packages/template-retail-react-app/app/pages/checkout/partials/shipping-options.jsx

```diff
@@ -218,8 +218,8 @@ export default function ShippingOptions() {
                 <ToggleCardSummary>
                     <Flex justify="space-between" w="full">
                         <Text>{selectedShippingMethod.name}</Text>
-                        <Flex alignItems="center" aria-label={shippingPriceLabel}>
-                            <Text fontWeight="bold" aria-hidden="true">
```

Comment:
Evinced complained that `aria-hidden` is incorrectly use without the `role` attribute. I dont think we should use `aria-hidden` here because it will be ignored by screen reader, the user will not be able to understand the shipping method price in this section.